framework/test-vectors/test-vector-enumeration.md

```diff
@@ -27,6 +27,14 @@ every possible input configuration for two reasons:
    and are not a priority to test at this time
    (see ["filtering input configurations"](#filtering-input-configurations))
 
+These reasons are both motivated by test efficiency.
+It is infeasible to enumerate every possible plaintext,
+or test every invalid encryption configuration.
+By strategically identifying test cases to enumerate,
+the test vector framework employs orders of magnitude of leverage
+to have comprehensive test coverage
+with a concise suite of tests.
```

Comment:
Well said.
Also, it leaves open the door for someone to manually construct their own test case to validate or explore the possibilities